docs/UserGuide.md

```diff
@@ -113,43 +88,9 @@ Format: `list`
 
 Expected outcome (success):\
 Here is the list of all contacts in your address book:
-1. John Doe 98765432 johnd@example.com John street, Block 123, #01-01
-2. Loysius 91234567 blizzerac@example.com Pasir Ris Avenue 6
-3. Jason Baek 97896054 jasonbaek@example.com Ivory tower 6
-
+1. Loysius 91234567 blizzerac@example.com Pasir Ris Avenue 6
+2. Jason Baek 97896054 jasonbaek@example.com Ivory tower 6
```

Comment:
Should these be replaced with details of the animals instead of humans?

Reply:
I will change it to details of the animals. However, whoever's implementing the function will have to write it in greater detail later on.